vagrant_repo/data_bags/cerner_splunk/multisiteUS.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "multisiteUS",
+  "sites" : ["cerner_splunk/cluster-vagrant-site1", "cerner_splunk/cluster-vagrant-site2"],
+  "settings": {
```

Comment:
Does this cluster come up clean with no warning/error messages?  Based on the [doc](http://docs.splunk.com/Documentation/Splunk/7.0.0/Indexer/Multisiteconffile#How_multisite_configuration_differs_from_single-site_configuration), since there are only 2 nodes in site2, I would expect it to fail to meet the default replication factor of 3.  Seems like we should set the replication_factor to 2 in this databag.